src/Main/Intro.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Email from '../assets/email-icon.png'
+import Gitter from '../assets/gitter-icon.png'
+
+const Intro = (props) => {
+  return (
+    <div className="intro">
```

Comment:
Can we make the line height of the text in this component bigger? Seems too close together